### PR TITLE
BZ#2188239 - better detect already migrated PostgreSQL data

### PR DESCRIPTION
**Pull request by evgeni**
Users can (and in some cases have to) manually move the PostgreSQL data to the new location. In the case when they would completely delete the old location, the facts code would previously error out:

    Traceback (most recent call last):
      File "/usr/lib64/python2.7/multiprocessing/process.py", line 258, in _bootstrap
        self.run()
      File "/usr/lib64/python2.7/multiprocessing/process.py", line 114, in run
        self._target(*self._args, **self._kwargs)
      File "/usr/lib/python2.7/site-packages/leapp/repository/actor_definition.py", line 72, in _do_run
        actor_instance.run(*args, **kwargs)
      File "/usr/lib/python2.7/site-packages/leapp/actors/init.py", line 289, in run
        self.process(*args)
      File "/usr/share/leapp-repository/repositories/system_upgrade/el7toel8/actors/satellite_upgrade_facts/actor.py", line 96, in process
        scl_psql_stat = os.stat(POSTGRESQL_SCL_DATA_PATH)
    OSError: [Errno 2] No such file or directory: '/var/opt/rh/rh-postgresql12/lib/pgsql/data/'

However, if the user already has migrated the data, we don't have to do it in the actor and thus can skip all that detection code.

Also adjust the report to say the data looks migrated (but still needs a reindex).

### CODE REVIEW

File: repos/system_upgrade/el7toel8/actors/satellite_upgrade_check/libraries/satellite_upgrade_check.py
Comment (fernflower) [inline repos/system_upgrade/el7toel8/actors/satellite_upgrade_check/libraries/satellite_upgrade_check.py:32]: Nit: Will it make sense to add "No additional action will be taken" to provide more clarity to the user?

Comment (evgeni) [reply repos/system_upgrade/el7toel8/actors/satellite_upgrade_check/libraries/satellite_upgrade_check.py:32]: Yes and no ;-)
An action *will* be taken: the reindex will happen. But you are right, no *movement* of the data will take place, which is a good thing to point out! Will add!

**Comment by evgeni** [comment]
> ` Failed loading plugin "copr": module 'dnf' has no attribute 'cli'`

this looks not like an related error?

**Comment by fernflower** [comment]
@evgeni It is unrelated, yeah, sorry about that. It has emerged recently and we are actively investigating it.
BTW you don't need to leave a '/rerun' comment anymore, now packit will schedule tests for you. I guess I need to update the PR bot message :)

**Comment by pirat89** [comment]
@lpramuk Hi Lukas, can you (or someone) test it before the merge and confirm it works as expected?

**Review by pirat89 — APPROVED** [review]
lgtm. waiting for @lpramuk for the feedback about the testing for now.

**Pull request closed without merging**
